packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -127,9 +131,15 @@ const trace = makeTracer('VM');
  * }>} MethodContext
  */
 
-// FIXME https://github.com/Agoric/agoric-sdk/issues/5622
-let liquidationQueueing = false;
-let outstandingQuote = null;
+/** @typedef {{
```

Comment:
I think this is the right place for a comment saying that these values are ephemeral, and per-vaultManager, and why that's the case.